src/PDF/TestPDFParser/TestPDFParser.py

```diff
@@ -0,0 +1,14 @@
+from src.Class.DocumentClass import Class
+from src.PDF.PDFParser import PDFParser
+from os import walk
+
+f = []
+for dir_path, dir_names, file_names in walk('C:\\Users\\Slava\\Downloads\\Telegram Desktop\\MagiRemoved'):
```

Comment:
Абсолютный путь к тестовым файлам - не лучшее решение, так как у других членов команды тесты не пройдут.
_____
The absolute path to the text files is not the best solution, since other team members will not pass the tests.